audobject/core/object.py

```diff
@@ -235,6 +235,36 @@ def resolvers(self) -> typing.Dict[str, resolver.Base]:
             return self.__dict__[define.CUSTOM_VALUE_RESOLVERS]
         return {}
 
+    @property
+    def short_id(self) -> str:
+        r"""Short object identifier.
+
+        The short ID consists of eight characters
+        and is created from its non-hidden arguments.
+
+        Returns:
+            short object identifier
+
+        Examples:
+            >>> class Foo(Object):
+            ...    def __init__(self, bar: str):
+            ...        self.bar = bar
+            >>> foo1 = Foo('I am unique!')
+            >>> print(foo1.id)
```

Comment:
We also do it in the docstring example of `Object.id`, but I wonder why we have to use `print()` here cannot simply do `foo1.id`?

Reply:
Good observation. There is no reason. I updated the examples for `short_id` and `id`.